assets/css/less/admin-settings.less

```diff
@@ -258,6 +258,12 @@
     }
 }
 
+/* Using the ID selector is necessary because there is no class-based identifier for this settings */
+.wpbdp-no-renewal #wpbdp-settings-admin-notifications .wpbdp-settings-multicheck-option-no-2,
```

Comment:
It's possible that this generic class on the checkboxes could cause issues in the future if we add or rearrange options. Maybe we could add another class in the setting_multicheck_callback() function? It could include the $option_value so we know we are hiding the right options.

Reply:
That's a good idea. On it.